packages/dashboard/src/views/Holdings.js

```diff
@@ -198,7 +216,9 @@ const Holdings = ({
       </Row>
 
       {/* Portfolio Quick Look */}
-      <Row>{accountSmallStats}</Row>
+      <Row>{accountSmallStats.slice(0,3)}</Row>
```

Comment:
In react you can render the whole array if its filled with JSX. If you want to omit an element like you did here, just get rid of it (or comment it out) from the initial data

Reply:
just realized youre doin this to get the data elems into diff rows. thats fine with me 👍